community/governance.md.tmpl

```diff
@@ -37,14 +37,20 @@ Here are defined the primary teams participating in conda-forge activities.
   core, part of staged-recipes, or maintainers. This includes first-time
   contributors, collaborators, and funders. They have no special rights within
   the conda-forge organization itself.
-* **emeritus-core:** Core members that are inactive (commits, GitHub comments/issues/reviews,
-  dev meetings and voting on polls) in the past six months will be asked if they want to become emeritus-core
-  developers. Any core member can also request to become emeritus if they wish to do so
-  (e.g. taking a sabbatical or long vacation).
-  Emeritus core members can still vote and move back to active core anytime. Emeritus
-  votes are used to count towards quorum but the quorum size is computed from the size of
-  the active core group. The `core.csv` list should be updated when a change in the status
-  of a member occurs.
+* **emeritus-core:** Core members that do not show any activity in their core
+responsibilities (including organization maintenance in the form of commits, GitHub comments, issues, or reviews; participation in chat platforms and community meetings; voting on polls) in the
+  past six months will be asked if they want to become emeritus-core developers. One
+  week after asking, if the inactive core member has not responded, they will be
```

Comment:
```suggestion
* **emeritus-core:** Core members that do not show any activity in their core
  responsibilities (including organization maintenance in the form of commits,
  GitHub comments, issues, or reviews; participation in chat platforms and
  community meetings; voting on polls) in the past six months will be 
  asked if they want to become emeritus-core developers. Four weeks after 
  asking, if the inactive core member has not responded, they will be
```

Reply:
I changed this delay period to four weeks from one week to match the period below.